cookbooks/active_dau.md

```diff
@@ -0,0 +1,30 @@
+# Active DAU
+
+An "Active User" is defined as a client who has total_daily_uri >= 5 URI for a given date. 
+* Dates are defined by `submission_date_s3`.
+* A client's total_daily_uri is defined as their sum of `scalar_parent_browser_engagement_total_uri_count` for a given date[1]. 
+
+Here is an example query in `main_summary` for getting Active DAU (aDAU) over time: 
+
+```sql
+SELECT
+    submission_date_s3, 
+    count(DISTINCT client_id) as aDAU
+FROM 
+    (SELECT 
+            submission_date_s3, 
+            client_id, 
+            sum(coalesce(scalar_parent_browser_engagement_total_uri_count, 0)) as total_daily_uri
+        FROM
+            main_summary
```

Comment:
I'm concerned readers will blindly copy/paste this query and bork re:dash. Can you restrict the `submission_date_s3` and `sample_id` range?